Runtime/Scripts/Plugins/AnimationPointer/MaterialPropertiesRemapper.cs

```diff
@@ -0,0 +1,485 @@
+﻿using System;
+using System.Collections.Generic;
+using GLTF.Schema;
+using UnityEngine;
+
+namespace UnityGLTF.Plugins
+{
+    public class MaterialPointerPropertyMap
+    {
+        public enum PropertyTypeOption {LinearColor, SRGBColor, Texture, LinearTexture, TextureTransform, Float}
```

Comment:
I think we want that to be on multiple lines for easier future changes

Reply:
Changed with 9de922c583e869c97aa2417986cb7240dca19752